### PR TITLE
Ec 14 translate column names

### PR DESCRIPTION
**Pull request by g1er**

### CODE REVIEW

File: src/app/cart/components/placed-orders/placed-orders.component.html [hunk 1]
```diff
@@ -18,48 +19,14 @@
           onerror="this.onerror=null;this.src='/assets/images/teapod.jpeg';" alt="image of {{placedOrderProductsDescription.product.name}}" style="max-width: 100px;"></td>
         <td class="align-middle">{{placedOrderProductsDescription.product.description}}</td>
         <td class="align-middle">$ {{placedOrderProductsDescription.product.price}}</td>
-        <td class="align-middle">{{placedOrderProductsDescription.quantity}} pcs</td>
+        <td class="align-middle">{{placedOrderProductsDescription.quantity}} {{'placedOrder.pcs' | translate}}</td>
         <td class="align-middle">$ {{ totalSum(placedOrderProductsDescription.product.price, placedOrderProductsDescription.quantity) }}</td>
-        <td class="align-middle"><button class="btn btn-danger">Cancel</button></td>
+        <td class="align-middle">01/11/2020</td>
```
Comment (IgorKulishov) [inline src/app/cart/components/placed-orders/placed-orders.component.html:24]: For your information:
Backend will provide timestamp of date in number format 1604286959399 and we will need to add logic on UI to format to DD/MM/YYYY

File: src/app/layout/header/components/header.component.scss [hunk 2]
```diff
@@ -30,9 +30,9 @@
   transform:rotate(90deg);
   border: none;
 }
-.open-menu-button:hover {
-  //background-image: linear-gradient(to right bottom, #ffffff, rgba(52, 152, 219, 0.5));
-}
+// .open-menu-button:hover {
```
Comment (IgorKulishov) [inline src/app/layout/header/components/header.component.scss:33]: Will we need to use the commented out code later?
If not, then it's better to remove code.

**Review by IgorKulishov — APPROVED** [review]

File: src/app/cart/components/placed-orders/placed-orders.component.html (hunk 1 above)
Comment (g1er) [reply src/app/cart/components/placed-orders/placed-orders.component.html:24]: How we can do it? With pipe?

File: src/app/layout/header/components/header.component.scss (hunk 2 above)
Comment (g1er) [reply src/app/layout/header/components/header.component.scss:33]: I think we don't need it. Actually it's not my code. I'll remove it and push in next PR

File: src/app/cart/components/placed-orders/placed-orders.component.html (hunk 1 above)
Comment (IgorKulishov) [reply src/app/cart/components/placed-orders/placed-orders.component.html:24]: Yes, we can do it with pipe
Here is doc https://docs.angularjs.org/api/ng/filter/date

File: src/app/layout/header/components/header.component.scss (hunk 2 above)
Comment (IgorKulishov) [reply src/app/layout/header/components/header.component.scss:33]: Yes, let's do clean up and remove any unused code or even imported libraries that are not used. That all impacts performance. If we need any code we can find in history.

**Review by IgorKulishov — APPROVED** [review]

**Pull request merged**